docs/imaging/how-to-add-sessions.rst

```diff
@@ -5,30 +5,35 @@ Imaging:  Web GUI
 ===========================
 
 1. Copy files to a file server
-2. Open the imaging `Website`_ and log in (ask Simon Ball or Horst Obenhaus for log in credentials if this is your first log in)
+2. Open the [Web GIU website](https://datajoint.kavli.org.ntnu.no/) and log in (ask Simon Ball or Horst Obenhaus for log in credentials if this is your first log in)
 
-    - Go to *BaseFolder* -> *Add BaseFolder*
+    - Go to *Imaging* -> *BaseFolders* -> *Add BaseFolder*
     
-3. Fill out the details in the mask on that website. Make sure you get the session logic *combined* right. It determines how sessions and recordings are created in the database. This only has relevance for data acquired with scanimage (miniscope, ...). For femtonics recordings this function is not implemented and has to remain on "no" (not combined). :ref:`Imaging ingestion combined`
+3. Fill out the details in the mask on that website. Make sure you get the session logic *combined* right. It determines how sessions and recordings are created in the database. This only has relevance for data acquired with scanimage (miniscope, ...). See :ref:`Imaging ingestion combined`. For femtonics recordings this function is not implemented and has to remain on "no" (not combined).
 
     - **Experimenter:** select your username from the list.
     
     - **File path:** specify the file path to where your raw data is stored.
     
-    - **Combined?:**  See :ref:`Imaging ingestion combined`. For the Femtonics 
+    - **Combined?:**  See :ref:`Imaging ingestion combined`. For the Femtonics this must be "no".
     
-    - **Setup:** Select which microscope type you used (Illuminato = Femtonics).
+    - **System Configuration:** Select which microscope type you used (Illuminato = Femtonics).
     
     - **Scope model:** for miniscope users only, select which model you used.
     
     - **Animal ID in mLIMS:** the animal ID of your furry collaborator as registered in mLIMS. 
     
-    - **Suite2P python options:** this selection determines how Suite2p, which is incorporated into the pipeline, analyses your data. You can make your own set of options that work best with your data, but for now just choose an existing one and see how that works. **Miniscope users:** choose an option that matches the indicator you're using. **Femtonics users:** flavio-develop is a good place to start. 
+    - **Suite2P python options:** this selection determines how Suite2p, which is incorporated into the pipeline, analyses your data. You can [make your own set of options](https://moser-pipelines.readthedocs.io/en/latest/imaging/basics.html#create-a-new-suite2p-options-file-for-your-data) that work best with your data, but for now just choose an existing one and see how that works first.
```

Comment:
Avoid linking absolute URLs where possible For external links, it can't be avoided, but for internal links, it can. RST supports link templating in the LaTeX style (which is the primary reason I switched): you add a link anchor to the destination, which might look like this:
```
If suite2p hasn't done a good job of identifying cells, it may be worth creating your own options file with settings tuned specifically to your data (see below).

.. _Imaging Basics Add-Suite2p-Options:

Create a new suite2p options file for your data
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
And the link itself is created like so:
```
:ref:`Imaging Basics Add-Suite2p-Options`
```

As long as the anchor still exists and doesn't change, this kind of link templating will then automatically update if the url structure changes for some reason.